src/arviz_plots/plots/distplot.py

```diff
@@ -250,6 +257,33 @@ def plot_dist(
                 **density_kwargs,
             )
 
+        elif kind == "hist":
+            # WIP
+            print("----------\n\n")
+            hist_dict = {}
+            # loops through the data variables in distribution and calls histogram() for each
+            for var_name in distribution.data_vars:
+                # print(f"    Var name: {var_name!r}\n")
+                var_data = distribution[var_name]
+                # print(f"    Var data: {var_data!r}\n")
+
+                # number of bins is set to 20 by default
+                hist = histogram(
+                    da=var_data, dims=density_dims, bins=20, **stats_kwargs.get("density", {})
```

Comment:
the default number of bins should be taken care of by `histogram` function. Moreover, if we believed a different default were needed, it would have to happen via `stats_kwargs` and `setdefault` so a user could modify that value.

Reply:
Just modified this in the latest commit to do so 

Now by default, with no bins provided (`azp.plot_dist(schools, kind='hist')`) and the same example dataset, this is produced: 
![distplot-hist-matplotlib](https://github.com/arviz-devs/arviz-plots/assets/24991131/84721a96-e8cc-445a-80de-29a9add1175a)

And with bins=40 provided by a user (`azp.plot_dist(schools, kind='hist', stats_kwargs={"density": {"bins":40}})`), this is produced:
![distplot-hist-matplotlib](https://github.com/arviz-devs/arviz-plots/assets/24991131/924f3e90-604b-4b19-b647-b15a3f72d5f7)

Will look at the other comments and make changes too now